.github/workflows/web-app-deployer.yml

```diff
@@ -16,6 +16,12 @@ on:
         description: Host name to be used in preview deployment
         type: string
         required: false
+      REACT_APP_GOOGLE_ANALYTICS_ID:
+        description: Google Analytics ID
+        type: string
+        # defaulting to a placeholder to keep the GA ID empty if parameter is not passed
+        default: '{{REACT_APP_GOOGLE_ANALYTICS_ID}}'
```

Comment:
This looks wrong. Where's the $

Reply:
This is the placeholder of the property file, [here](https://github.com/MobilityData/mobility-feed-api/blob/bebf0208bc8d1a4e1f4de72be1365bb35b5ad42c/web-app/src/.env.rename_me#L9). So the value will be kept as the placeholder, this makes the environment to be empty at runtime, implementation [here](https://github.com/MobilityData/mobility-feed-api/blob/bebf0208bc8d1a4e1f4de72be1365bb35b5ad42c/web-app/src/app/utils/config.ts#L7)

Reply:
Keeping the placeholder, let's keep local development and testing and ignore the value. The pattern `key={{key}}` is consistently use in the configuration file. Having a "NO_VALUE" is also an option, but requires that the develop[per actually add the value to the property file. It's easier to keep the property file unmodified and get empty string.